src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php

```diff
@@ -41,69 +39,62 @@ final class DashboardController
     /** @var SalesDataProviderInterface|null */
     private $salesDataProvider;
 
+    /** @var StatisticsDataProviderInterface */
+    private $statisticsDataProvider;
+
     public function __construct(
-        DashboardStatisticsProviderInterface $statisticsProvider,
         ChannelRepositoryInterface $channelRepository,
         EngineInterface $templatingEngine,
         RouterInterface $router,
-        ?SalesDataProviderInterface $salesDataProvider = null
+        ?SalesDataProviderInterface $salesDataProvider = null,
+        ?StatisticsDataProviderInterface $statisticsDataProvider = null
     ) {
-        $this->statisticsProvider = $statisticsProvider;
         $this->channelRepository = $channelRepository;
         $this->templatingEngine = $templatingEngine;
         $this->router = $router;
         $this->salesDataProvider = $salesDataProvider;
-
-        if ($this->salesDataProvider === null) {
```

Comment:
This note should be brought back IMHO. Why did you remove it? Similar note should be added for `statisticsDataProvider`